commec/config/screen_databases.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
+"""
+Defines the `Input and Output Screen Parameters` class, and associated dataclasses.
+"""
+import os
+from commec.config.io_parameters import ScreenIOParameters
+from commec.tools.blastn_handler import BlastNDataBase
+from commec.tools.blastx_handler import BlastXDataBase
+from commec.tools.blastdmnd_handler import DiamondDataBase
+from commec.tools.cmscan_handler import CmscanDataBase
+from commec.tools.hmm_handler import HMMDataBase
+
+class CommecDatabases():
+    """
+    Consolidation and initialisation of all databases needed for the commec workflow.
+    This could be maybe split into a Biorisk_handler, Taxonomy handler, and Benign handler
+    class objects respectively.
+    """
+    def __init__(self, params : ScreenIOParameters):
+        # Biorisk
+        self.biorisk_db : HMMDataBase = None
+        # Taxonomy
+        self.protein_db : BlastXDataBase = None
+        self.nucleotide_db : BlastNDataBase = None
+        # Benign
+        self.benign_hmm : HMMDataBase = None
+        self.benign_blastn : BlastNDataBase = None
+        self.benign_cmscan : CmscanDataBase = None
+
+        if params.should_do_biorisk_screening:
+            self.biorisk_db = HMMDataBase(
+                os.path.join(params.db_dir, "biorisk_db"),
+                os.path.join(params.db_dir, "biorisk_db/biorisk.hmm"),
+                params.query.fasta_aa_filepath,
+                f"{params.output_prefix}.biorisk.hmmscan",
+            )
+
+        if params.should_do_protein_screening:
+            if params.inputs.search_tool == "blastx":
```

Comment:
This further makes me think  `inputs` should be called `run_config` or `screen_config`

Reply:
Hmmm I do like the use of input somewhere, as these are specifically those configurations able to be input from the user... Not wedded to the nomenclature, I suppose config works in a similar vein.

Reply:
renamed to config in latest version.